src/utils/HTTPTransport.ts

```diff
@@ -0,0 +1,92 @@
+export enum Method {
+  Get = 'Get',
+  Post = 'Post',
+  Put = 'Put',
+  Patch = 'Patch',
+  Delete = 'Delete'
+}
+
+type Options = {
+  method: Method;
+  data?: any;
```

Comment:
важной часть протокола HTTP являются заголовки, их тоже стоит задавать в ```options```, а тип ```any``` лучше заменить на ```unknown```
```suggestion
  data?: unknown;
  headers: Record<string, string>
```